test/parallel/test-fs-promises.js

```diff
@@ -38,6 +39,8 @@ const tmpDir = tmpdir.path;
 
 common.crashOnUnhandledRejection();
 
+assert.strictEqual(Object.keys(fs).includes('promises'), false);
+
```

Comment:
A comment noting that this is needed only while and because `.promises` is experimental might help for future maintenance.